tests/scripts/cleanup-gcp-pipeline.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+source scripts/vars.env
+
+gcloud compute instances delete ${RESOURCE_NAME} --quiet --project=${GKE_PROJECT} --zone=${GKE_CLUSTER_ZONE}
+gcloud compute firewall-rules delete ${RESOURCE_NAME} --quiet --project=${GKE_PROJECT}
+
+gcloud container clusters delete ${GKE_CLUSTER_NAME} --zone ${GKE_CLUSTER_ZONE} --project ${GKE_PROJECT} --quiet
```

Comment:
Is this script needed when we have other scripts that already do this cleanup?

Reply:
The other script attempts to delete the router, which we are not creating in the pipeline

Reply:
The router script is separate from the VM script.

Reply:
In that case I can't remember 😅 let me look

Reply:
Ah, it was around cleaning up the master authorised networks, which isn't required in the pipeline as we are not keeping the cluster. I've changed the cleanup-vm script to accept a conditional, so if we are running in the pipeline we skip that step.